src/EditorFeatures/Core/Classification/Syntactic/SyntacticClassificationTaggerProvider.cs

```diff
@@ -2,43 +2,47 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.ComponentModel.Composition;
-using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
-using Microsoft.CodeAnalysis.Editor;
 using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
 using Microsoft.CodeAnalysis.Editor.Shared.Tagging;
 using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
 using Microsoft.CodeAnalysis.Editor.Tagging;
 using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Shared.TestHooks;
 using Microsoft.VisualStudio.Text;
-using Microsoft.VisualStudio.Text.Editor;
 using Microsoft.VisualStudio.Text.Tagging;
-using Microsoft.VisualStudio.Utilities;
 
 namespace Microsoft.CodeAnalysis.Classification
 {
-    [Export(typeof(ITaggerProvider))]
-    [ContentType(ContentTypeNames.RoslynContentType)]
-    [TextViewRole(PredefinedTextViewRoles.Document)]
-    [TagType(typeof(IClassificationTag))]
-    [method: ImportingConstructor]
-    [method: SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification = "Used in test code: https://github.com/dotnet/roslyn/issues/42814")]
+    /// <summary>
+    /// Intentionally not exported.  It is consumed by the <see cref="TotalClassificationTaggerProvider"/> instead.
+    /// </summary>
     internal partial class SyntacticClassificationTaggerProvider(
         IThreadingContext threadingContext,
-        SyntacticClassificationTypeMap typeMap,
+        ClassificationTypeMap typeMap,
         IGlobalOptionService globalOptions,
         IAsynchronousOperationListenerProvider listenerProvider) : ITaggerProvider
     {
         private readonly IAsynchronousOperationListener _listener = listenerProvider.GetListener(FeatureAttribute.Classification);
         private readonly IThreadingContext _threadingContext = threadingContext;
-        private readonly SyntacticClassificationTypeMap _typeMap = typeMap;
+        private readonly ClassificationTypeMap _typeMap = typeMap;
         private readonly IGlobalOptionService _globalOptions = globalOptions;
 
         private readonly ConditionalWeakTable<ITextBuffer, TagComputer> _tagComputers = new();
 
-        public ITagger<T>? CreateTagger<T>(ITextBuffer buffer) where T : ITag
+        ITagger<T>? ITaggerProvider.CreateTagger<T>(ITextBuffer buffer)
```

Comment:
>ITagger<T>? ITaggerProvider.CreateTagger<T>(ITextBuffer buffer) [](http://example.com/codeflow?start=8&length=63)

Not part of this PR, but I'm curious why this uses a CWT instead of the buffer properties to store the TagComputer

Reply:
that is a wonderful question...

Reply:
@jasonmalinowski  do you recall any reason for this?  I'm happy to change this in a followup pr.

Reply:
handlign in a followup PR.

Reply:
I believe the only reason for CWTs was because some folks were scared to use buffer properties because it means somebody could _theoretically_ grab the objects out of the property bags.